src/axom/sidre/core/Group.cpp

```diff
@@ -1340,19 +1340,33 @@ void Group::copyToConduitNode(Node& n) const
   while(indexIsValid(vidx))
   {
     const View* view = getView(vidx);
-    Node& v = n["views"].fetch(view->getName());
-    view->copyToConduitNode(v);
-
+    if(isUsingMap())
+    {
+      Node& v = n["views"].fetch(view->getName());
+      view->copyToConduitNode(v);
+    }
+    else
+    {
+      Node& v = n["views"].append();
+      view->copyToConduitNode(v);
```

Comment:
Quick clarification -- how does this work?

My assumption is that it's appending an empty node `v` to `n["views"] and then `copyToConduitNode(v)` fills in `v`?
Is that right?

Reply:
Looks like it:
https://github.com/LLNL/conduit/blob/de3a873068ba971405cfc003d3286673f2cedfcd/src/libs/conduit/conduit_node.hpp#L3971-L3975

Reply:
Yes, your description is correct.